parts/k8s/artifacts/kubernetesazurekms.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=azurekms
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=simple
+Restart=always
+EnvironmentFile=-/etc/default/%p
+TimeoutStartSec=0
```

Comment:
That is a bit too low isn't?

Reply:
`Pass "0" to disable the timeout logic.` https://manpages.debian.org/jessie/systemd/systemd.service.5.en.html